source/release-notes.txt

```diff
@@ -10,6 +10,36 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.45.1
+----------------
+
+*Released January 15, 2024*
+
+New Features:
+
+- Adds welcome page (:issue:`COMPASS-8657`)
+- Adds onFailToLoadConnections callback (:issue:`COMPASS-8657`)
+
+Bug Fixes:
+
+- Send a nonce in oidc request by default (:issue:`COMPASS-8588`)
+- Don't cut off list view value editor tooltips (:issue:`COMPASS-8581`)
+- Fixes element key editor tooltip and revert not updating validity 
+  (:issue:`COMPASS-8586`)
+- Don't fallback to zero if the count cannot be determined
+- Updates top chart write lock percentage of load visual 
+  (:issue:`COMPASS-8604`)
+- Don't lose the isOperationInProgress state when the user switches 
+  tabs (:issue:`COMPASS-8576`)
+- Bumps mongosh to 2.3.6 (:issue:`COMPASS-8689`)
+- Don't derive enableGenAIFeatures from org, allow disabling it 
+  (:issue:`COMPASS-8680`)
+- Adjusts status marker position (:issue:`CLOUDP-289203`)
```

Comment:
[question for tech reviewer]

Just double-checking: is CLOUDP-289203 the correct issue # here? Thanks!